packages/core/src/fhir-hydration/resources/observation.ts

```diff
@@ -0,0 +1,91 @@
+import { Observation, Quantity } from "@medplum/fhirtypes";
+import convert, { Unit } from "convert-units";
+
+/**
+ * This map is used to normalize unconventional unit names to standard unit names. i.e cel -> C, millimeter to m, etc.
+ */
+const unitNormalizationMap = new Map<string, string>([["cel", "C"]]);
+
+/**
+ * This map is used to convert standard units to preferred units. i.e C -> F, g -> kg, etc.
+ */
+const unitConversionMap = new Map<string, string>([
+  ["C", "F"],
+  ["g", "kg"],
+  ["lb", "kg"],
+  ["m", "cm"],
```

Comment:
what is actually preferred here we probably need to ask customers

Reply:
yea, this is a draft, but def need to double check some units!

Reply:
brought this up in [this thread](https://metriport.slack.com/archives/C06CQC7GQG3/p1729657279660259?thread_ts=1729629627.073189&cid=C06CQC7GQG3)